packages/augur-sdk/src/api/Liquidity.ts

```diff
@@ -152,11 +152,11 @@ export class Liquidity {
 
       // BIDS
       for (let outcome = 1; outcome < numOutcomes; outcome++) {
```

Comment:
need to add comment for future. outcome starts at 1 because invalid is outcome 0 and not included in liquidity calc. something like that.